pkg/apis/kops/componentconfig.go

```diff
@@ -854,6 +854,12 @@ type NodeTerminationHandlerConfig struct {
 
 	// EnablePrometheusMetrics enables the "/metrics" endpoint.
 	EnablePrometheusMetrics *bool `json:"prometheusEnable,omitempty"`
+
+	// EnableSqsTerminationDraining enables queue-processor mode which drains nodes when an SQS termination event is received.
+	EnableSqsTerminationDraining *bool `json:"enableSqsTerminationDraining,omitempty"`
+
+	// ManagedAsgTag is the tag used to determine which nodes NTH can take action on
+	ManagedAsgTag *string `json:"managedAsgTag,omitempty"`
```

Comment:
I think we should use consistent casing for these new fields, to be consistent with other API fields, for example in ClusterSpec: `API`, `IAM`, `AWSLoadBalancerController`, `ExternalDNS`, `SSHAccess`, etc.

I'm curious for others' thoughts here though.

```suggestion
	// EnableSQSTerminationDraining enables queue-processor mode which drains nodes when an SQS termination event is received.
	EnableSQSTerminationDraining *bool `json:"enableSQSTerminationDraining,omitempty"`

	// ManagedASGTag is the tag used to determine which nodes NTH can take action on
	ManagedASGTag *string `json:"managedASGTag,omitempty"`
```